Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.5
```

Comment:
**suggestion**: WDYT about moving the docker-related stuff to a dedicated subdir `docker/`? This layout comes handy when there's more docker-related configuration required, and changing it in the future might be cumbersome.

Reply:
Makes sense.